src/test/PlanetTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace PlanetApp\Test;
+use PHPUnit\Framework\TestCase;
+use PlanetApp\Planet;
+
+class PlanetTest extends TestCase
+{
+    public function testSuccessCreateCardHtml()
+    {
+        $Planet = new Planet();
+        $Planet->setName('Uranus');
+        $Planet->setType('Ice giant');
+        $Planet->setImageSrc('pretend-url');
+
```

Comment:
variable name should be lowercase - $planet

Reply:
done